server/src/test/java/org/candlepin/resource/HypervisorResourceTest.java

```diff
@@ -127,12 +127,16 @@ public void setupTest() {
         this.i18n = I18nFactory.getI18n(getClass(), Locale.US, I18nFactory.FALLBACK);
 
         this.hypervisorType = new ConsumerType(ConsumerTypeEnum.HYPERVISOR);
+
         this.hypervisorType.setId("test-hypervisor-ctype");
 
         this.mockConsumerType(this.hypervisorType);
 
         this.translator = new StandardTranslator(this.consumerTypeCurator);
 
+        when(consumerTypeCurator.lookupByLabel(eq(ConsumerTypeEnum.HYPERVISOR.getLabel())))
```

Comment:
Another rebasing issue here. Pretty sure this is handled or the mocking needs to be changed. The existing code in master should work for this.